foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -46,29 +46,56 @@ Follow these steps to add the install the extension in Experience Platform Launc
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 Add the library to your project via your [Cocoapods](https://cocoapods.org/pods/AEPAssurance) `Podfile`
 
 ```text
-pod 'ACPCore'
+pod 'AEPCore'
 pod 'AEPAssurance'
 ```
 
-Import the Project Griffon libraries along with other SDK libraries:
+Import the Assurance extension along with the other Adobe Mobile extensions::
+
+#### Swift
+
+```swift
+import AEPCore
+import AEPAssurance // <-- import the AEPAssurance library
+```
 
 #### Objective-C
 
 ```objectivec
-#import "ACPCore.h"
-#import "AEPAssurance.h" // <-- import the AEPAssurance library
+@import AEPCore;
+@import AEPAssurance; // <-- import the AEPAssurance library
 ```
 
+{% endtab %}
+
+{% tab title="iOS (AEP 1.x)" %}
```

Comment:
Just to confirm - is AEP 1.x correct here?

Reply:
this is correct